bcl/src/bc_esp8266.c

```diff
@@ -378,6 +396,18 @@ static void _bc_esp8266_task(void *param)
                     sprintf(self->_command, "AT+CWJAP_CUR=\"%s\",\"%s\"\r\n", self->_config.ssid, self->_config.password);
```

Comment:
I have checked the buffer sizes, so it is safe to say buffer overflow will not happen here. On the other hand, I generally almost always prefer using `snprintf` to provide extra safety. Just something to consider, no change requested here...